src/api/middleware/validators/offer.js

```diff
@@ -69,6 +69,8 @@ const publishEndDateLimit = (publishEndDateCandidate, { req }) => {
     return true;
 };
 
+const normalizeDate = (date, _context) => (new Date(Date.parse(date))).toISOString();
```

Comment:
Why do you need a `_context` argument? It's not being used, it seems. Also, move this to the validators helpers file (middleware/validators/validatorUtils)

Reply:
This function is called by the validation/sanitization chain with 2 args, afaik, so I included 2 arguments in the function prototype, even though one of them is never used. I'm not sure if calling a function passing more arguments than those declared would cause problems, if not then I'll remove this argument.

Reply:
JS is pretty forgiving on that, it only matters that the order is correct (if you wanted to use the second arg, you'd need to specify the first, even though you wouldn't use it, so that js knows to assign the second one to the second variable) In this case, since you only use the first arg, you only need to specify the first one.

Additionally, as a side note, if a function has a signature with 3 arguments, you may call it with 1 or 2 parameters, and the others will be `undefined`. This is how you deal with "optional" arguments. This only works for the last arguments for obvious reasons